packages/translate/lib/__tests__/testLines.ts

```diff
@@ -5,6 +5,7 @@ import {
 	Snippets, StringComparator,
 	TestLine,
 } from '@suitest/types';
+import {PSVideoHaNoErrorCondition} from '@suitest/types/lib';
```

Comment:
should it be PSVideoHadNoErrorCondition ?

Reply:
Right, a typo.